inference-engine/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/strided_slice.cpp

```diff
@@ -11,90 +11,64 @@ using namespace LayerTestsDefinitions;
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 2, 2 }, { 1, 1, 1, 1 }, { 2, 2, 2, 2 }, { 1, 1, 1, 1 },
-                                {0, 0, 0, 0}, {1, 1, 1, 1},  {},  {},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 2, 2 }, { 1, 1, 1, 1 }, { 2, 2, 2, 2 }, { 1, 1, 1, 1 },
-                                {0, 0, 0, 0}, {0, 0, 0, 0},  {},  {},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 4, 3 }, { 0, 0, 0, 0 }, { 2, 2, 4, 3 }, { 1, 1, 2, 1 },
-                                {1, 1, 1, 1}, {1, 1, 1, 1},  {},  {},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 4, 2 }, { 1, 0, 0, 1 }, { 2, 2, 4, 2 }, { 1, 1, 2, 1 },
-                                {0, 1, 1, 0}, {1, 1, 0, 0},  {},  {},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 1, 2, 4, 2 }, { 1, 0, 0, 0 }, { 1, 2, 4, 2 }, { 1, 1, -2, -1 },
-                                {1, 1, 1, 1}, {1, 1, 1, 1},  {},  {},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 4, 2 }, { 1, 0, 0, 0 }, { 1, 2, 4, 2 }, { 1, 1, -2, -1 },
-                                {0, 1, 1, 1}, {1, 1, 1, 1},  {},  {},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 2, 2 }, { 0, 0, 0, 0 }, { 2, 2, 2, 2 }, { 1, 1, 1, 1 },
-                                {1, 1, 1, 1}, {1, 1, 1, 1},  {},  {},  {},
-                                InferenceEngine::Precision::I64, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 2, 2 }, { 1, 1, 1, 1 }, { 2, 2, 2, 2 }, { 1, 1, 1, 1 },
-                                {0, 0, 0, 0}, {1, 1, 1, 1},  {},  {},  {},
-                                InferenceEngine::Precision::I64, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 2, 2 }, { 1, 1, 1, 1 }, { 2, 2, 2, 2 }, { 1, 1, 1, 1 },
-                                {0, 0, 0, 0}, {0, 0, 0, 0},  {},  {},  {},
-                                InferenceEngine::Precision::I64, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 3, 4, 5, 6 }, { 0, 1, 0, 0, 0 }, { 2, 3, 4, 5, 6 }, { 1, 1, 1, 1, 1 },
-                                {1, 0, 1, 1, 1}, {1, 0, 1, 1, 1},  {},  {0, 1, 0, 0, 0},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
+InferenceEngine::Precision precision[] = {InferenceEngine::Precision::FP32, InferenceEngine::Precision::I64};
+
+stridedSliceSpecificParams ss_only_test_cases[] = {
```

Comment:
Did you remove some test cases?

Reply:
No, these are the same. I removed three identical tests which differed only in precision.